widget/entry_test.go

```diff
@@ -1736,6 +1736,19 @@ func TestPasswordEntry_ActionItemSizeAndPlacement(t *testing.T) {
 	assert.Equal(t, fyne.NewPos(e.MinSize().Width-2*theme.Padding()-b.Size().Width, 2*theme.Padding()), b.Position())
 }
 
+func TestPasswordEntry_Disabled(t *testing.T) {
+	entry, window := setupPasswordTest(t)
+	defer teardownImageTest(window)
```

Comment:
Do you really need to do this when not using any image tests?

BTW: I noticed that that function is writing to the same markup test file on each call which seems unnecessary. Will push a new PR to fix that.